docs/install.md

```diff
@@ -1,38 +1,64 @@
 # Installation
 
-We recommend **the easy way** if you are running your own server.
+There are many options for installing Sandstorm with various trade-offs. Choose the one that is most comfortable for you.
 
-## Installing the Easy Way
+Sandstorm requires Linux x86_64, with kernel version 3.13 or later.
 
-*Prerequisite:* Linux x86_64, with kernel version 3.13 or later.
+## Option 1: HTTPS-verified install
 
-To install on your own Linux machine, see
-[https://install.sandstorm.io/](https://install.sandstorm.io) or run:
+The easiest way to install Sandstorm is by running:
 
 ```bash
-    curl https://install.sandstorm.io | bash
+curl https://install.sandstorm.io | bash
 ```
 
-Note: If installing Sandstorm under LXC / Docker, you will need to choose the option to
-install as a non-root user. Unfortunately, this means the development tools will not
-work. This is due to the interaction between Sandstorm and Docker's use of Linux
-containerization features and missing features in the Linux kernel which we
-hope will be fixed eventually. For non-development purposes, Sandstorm should run just fine
-under Docker.
+## Option 2: Github-verified install
```

Comment:
BTW `raw.githubusercontent.com` is on Fastly, so (hilariously) this is just the choice of one CDN (github+fastly) vs. another CDN (sandstorm's GCE machines + CloudFlare). Whatever, though.